### ISSUE TITLE
Other models for recommendations?

### ISSUE THREAD

**Issue by ChandraPrakash-Bathula** [ChandraPrakash-Bathula/Movie-Recommendation-System#1]
Are there any other ml models better performing than these SVD Model, KNN Model,XGBoost Regression ?

**Comment by ChandraPrakash-Bathula** [comment]
Random Forest
Gradient Boosting Machines (e.g., XGBoost)
Neural Networks (Deep Learning)
Support Vector Machines (SVM)
k-Nearest Neighbors (KNN)
Naive Bayes
Decision Trees
Logistic Regression
AdaBoost
LSTMs (Long Short-Term Memory) for sequential data
